device/jsonld-contexts/device.jsonld

```diff
@@ -9,7 +9,6 @@
         "chargerStatus": "https://vocab.egm.io/chargerStatus",
         "cpuTemperature": "https://vocab.egm.io/cpuTemperature",
         "devEUI": "https://vocab.egm.io/devEUI",
-        "deviceType": "https://vocab.egm.io/deviceType",

```

Comment:
Maybe we should leave this for now and delete it later when I will have make the change for all existing entities